Source/Core/Core/HW/EXI/EXI_Channel.h

```diff
@@ -106,6 +109,9 @@ class CEXIChannel
   UEXI_CONTROL m_control;
   u32 m_imm_data = 0;
 
+  // storing this here is a bit silly but we'll try this for now...
```

Comment:
This should probably have a more informative comment explaining why it's silly and what the ideal scenario would be for where it should be located.